### PR TITLE
(DOCSP-27195): Correct information about some App Services limitations

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

I addressed all of the concerns outlined in the Jira ticket except for the Rules section. As noted there, the Rules section will be updated by other existing work.

### Jira

- https://jira.mongodb.org/browse/DOCSP-27195

### Staged Changes

- [Service Limitations](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-27195/reference/service-limitations/#rules)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [ n/a ] Create Jira ticket for corresponding docs-realm update(s), if any
- [ n/a ] Checked/updated Admin API
- [ n/a ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### Animal in a hat

<image src="https://ih0.redbubble.net/image.82683872.7091/flat,1000x1000,075,f.jpg" width="350">

### CODE REVIEW

File: source/reference/service-limitations.txt [hunk 1]
```diff
@@ -74,7 +74,8 @@ Time Series Collections
 
 :manual:`Time series collections </core/timeseries-collections/>` do not
 yet support :manual:`change streams </changeStreams/>`. Therefore,
-you cannot define triggers on or sync to a time series collection.
+you cannot define triggers on a time series collection. You can sync a time series
+collection, but the collection must be :ref:`asymmetrically synced <stream-data-from-client-to-atlas>`.
```
Comment (mongodben) [inline source/reference/service-limitations.txt:78]: rather than just using 'sync' as verb here, i think it should be explicit that you are syncing using Atlas Device Sync.

smthn like: 

```rst
You can only use a time series collection with Atlas Device Sync
using :ref:`Asymmetric Sync <stream-data-from-client-to-atlas>`.
```

**Review by mongodben — COMMENTED** [review]
looks good, 1 note abt rephrasing

File: source/reference/service-limitations.txt [hunk 2]
```diff
@@ -217,9 +222,10 @@ Sync - Concurrent Workloads
 ---------------------------
 
 Writing:
-  App Services supports up to 30 users concurrently writing data to an
-  underlying MongoDB document. With more than 30 concurrent writers, you
-  may see delays in syncing or conflict resolution.
+  For imoproved performance, you may want to limit the number of concurrent writes
```
Comment (cbush) [inline source/reference/service-limitations.txt:225]:
```suggestion
  For improved performance, you may want to limit the number of concurrent writes
```

File: source/reference/service-limitations.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/reference/service-limitations.txt]: Great suggestion! I riffed a bit on your suggestion.
``` rst
Atlas Device Sync supports time series collections through 
:ref:`Asymmetric Sync <stream-data-from-client-to-atlas>`.
```

Comment (mongodben) [reply source/reference/service-limitations.txt]: i don't think this is clear enough that you can only use time series collections if you are using Device Sync with Asymmetric Sync. could you rephrase to make it more explicit?

Comment (krollins-mdb) [reply source/reference/service-limitations.txt]: @mongodben, yeah, that makes sense. I expanded a bit to make it more explicit.

``` rst
You can use time series collections with Atlas Device Sync, but only 
:ref:`Asymmetric Sync <stream-data-from-client-to-atlas>` is supported. You can't 
use time series collections with Flexible Sync or Partition-Based Sync.
```

Comment (mongodben) [reply source/reference/service-limitations.txt]: i'd drop the 2nd sentence. Asymmetric Sync is still using Flexible Sync.

**Review by mongodben — APPROVED** [review]
lgtm. though please remove sentence as noted in the comment

File: source/reference/service-limitations.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/reference/service-limitations.txt]: Oh, right! It's just one-way. Thanks for the reminder.
